src/rpc/mempool.cpp

```diff
@@ -0,0 +1,206 @@
+            + HelpExampleCli("signrawtransaction", "\"myhex\"") +
+            "\nTest acceptance of the transaction (signed hex)\n"
+            + HelpExampleCli("testmempoolaccept", "\"signedhex\"") +
+            "\nAs a json rpc call\n"
+            + HelpExampleRpc("testmempoolaccept", "[\"signedhex\"]")
+            // clang-format on
+            );
+    }
+
+    RPCTypeCheck(request.params, {UniValue::VARR, UniValue::VBOOL});
+    const UniValue raw_transactions = request.params[0].get_array();
+    if (raw_transactions.size() < 1 || raw_transactions.size() > MAX_PACKAGE_COUNT) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER,
+                            "Too many transactions in package." );
+    }
+
+    Package package;
+    for (const auto& rawtx : raw_transactions.getValues()) {
+        CMutableTransaction mtx;
+        if (!DecodeHexTx(mtx, rawtx.get_str())) {
+            throw JSONRPCError(RPC_DESERIALIZATION_ERROR,
+                                "TX decode failed: " + rawtx.get_str() + " Make sure the tx has at least one input.");
+        }
+        package.push_back(MakeTransactionRef(std::move(mtx)));
+    }
+
+    if(package.size() != raw_transactions.size()){
+            throw JSONRPCError(RPC_INVALID_PARAMETER,
+                    "Duplicate transaction found in package.");
+    }
+
+    CAmount max_raw_tx_fee = ::maxTxFee;
+    if (!request.params[1].isNull() && request.params[1].get_bool()) {
+        max_raw_tx_fee = 0;
+    }
+
+    CValidationState state;
+    PackageValidationState pkg_results;
+    std::vector<CTxMemPoolEntry> submitPool;
+    submitPool.reserve(package.size());
+
+    bool success = false;
+    success = TestPackageAcceptance(package, state, pkg_results, &submitPool);
+
+    UniValue result(UniValue::VOBJ);
+
+    encodePackageResult(success, pkg_results, state, result);
+
+    return result;
+}
+
+static UniValue submitpackage(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
+    throw std::runtime_error(
+        "submitpackage \"[rawtx1, rawtx2]\" \n"
+        "\nSubmit a package of raw transactions (serialized, hex-encoded) to local node \n"
+        "\nArguments:\n"
+        "1. transaction_hex                        (array, required) An array of hex strings of raw transactions.\n"
+        "2. allowhighfees                            (boolean, optional, default=false) Allow high fees\n"
+        "\nResult:\n"
+        "   {\n"
+        "    \"allowed\"               (boolean) If submitting all transaction was successful \n"
+        "   }\n"
+        "\nExamples:\n"
+        "\nCreate a transaction\n"
+        + HelpExampleCli("createrawtransaction", "\"[{\\\"txid\\\" : \\\"mytxid\\\",\\\"vout\\\":0}]\" \"{\\\"myaddress\\\":0.01}\"") +
+        "Sign the transaction, and get back the hex\n"
+        + HelpExampleCli("signrawtransaction", "\"myhex\"") +
+        "\nSubmit the transaction as a package (signed hex)\n"
```

Comment:
There are various ways to create transactions, so `createrawtransaction` and `signrawtransaction` description are not necessary.